react-commenting.js

```diff
@@ -59,13 +78,22 @@ class GoodRobot extends Component{
   render(){
     return(
       // 8.
+      
+      //creates a division on the page where certain elements can be displayed.
```

Comment:
✔️ yes, and it is the single element that react will render on the page. it holds all other dom elements.